x-pack/plugins/rollup/public/crud_app/services/api.js

```diff
@@ -30,6 +31,8 @@ export async function deleteJobs(jobIds) {
 }
 
 export async function createJob(job) {
+  // Note that this will record an acion, even if the related request fails.
+  trackUserAction('create-rollup-job');
```

Comment:
@yaronp68 The way this is implemented will result in this action being tracked even if the "create job" request fails, e.g. server-side validation rejects it. This will mean we'll see a discrepancy between the number of user actions to create a rollup job, and the number of rollup jobs themselves. Will this be a problem?

Reply:
@cjcenizal I think that we are not trying to measure GUI failures vs. success and so having this "over-counting" is not desirable. However, If there's no good way to filter failures, it's not the end of the world as we will assume that usually the number represents the order of magnitude of using this feature (it doesn't really matter if there were 43 or 47 rollup jobs) and failures are marginal.

Reply:
@cjcenizal why not indeed only track success requests?

```js
return await getHttp().put(`${apiPrefix}/create`, body)
    .then((response) => {
        trackUserAction('create-rollup-job');
        return response;
    })
```